lib/sarif/base_sarif.rb

```diff
@@ -135,11 +135,13 @@ def build_conversion
     def sarif_level(severity)
       case severity
       when "LOW"
-        SARIF_WARNINGS[:warning]
```

Comment:
This will map LOW severities to Sarif note across all adapters.  For most adapters that will result in SARIF with too low of a severity.

From the SARIF spec:
```
3.19.7 level property
A result object MAY contain a property named level whose value is one of a fixed set of strings that specify the severity level of the result.

If present, the level property SHALL have one of the following values, with the specified meanings:

·         "pass": The rule specified by the ruleId property (§3.19.4) was evaluated, and no problem was found.
·         "warning": The rule specified by the ruleId property was evaluated, and a problem was found.
·         "error": The rule specified by the ruleId property was evaluated, and a serious problem was found.
·         "open": The rule specified by the ruleId property was evaluated, and the tool concluded that there was insufficient information to decide whether a problem exists.
·         "notApplicable": The rule specified by the ruleId property was not evaluated, because it does not apply to the analysis target.
```
If you find it useful here's the gosec implementation for example:
https://github.com/securego/gosec/blob/6c57ae1628c42ebf2de515fd3f079477e0dc4f21/output/sarif_format.go#L169-L184

We'd deviate from that here. 

If you're finding that NPM low needs to be note for SARIF (or generally that the severity mappings need to vary per adapter which is a fair assumption) then you'll need to provide different mappings per adapter.

Reply:
We may want to include spec coverage across each adapter for all mappings too. I'm assuming the change from LOW->warning to LOW->note impacted the other scanners and ideally we would see specs fail w/that change.